src/getReleaseNotes.ts

```diff
@@ -79,33 +95,52 @@ async function getMonorepoReleaseNotes(
   repoUrl: string,
   workspaceRoot: string,
   rootManifest: MonorepoPackageManifest,
+  versioningStrategy: string,
 ): Promise<string> {
   const workspaceLocations = await getWorkspaceLocations(
     rootManifest.workspaces,
     workspaceRoot,
   );
 
   let releaseNotes = '';
-  for (const workspaceLocation of workspaceLocations) {
-    const completeWorkspacePath = pathUtils.join(
-      workspaceRoot,
-      workspaceLocation,
-    );
 
-    const rawPackageManifest = await getPackageManifest(completeWorkspacePath);
-    const { name: packageName, version: packageVersion } =
-      validatePolyrepoPackageManifest(
-        rawPackageManifest,
+  if (versioningStrategy === FIXED) {
+    for (const workspaceLocation of workspaceLocations) {
+      const completeWorkspacePath = pathUtils.join(
+        workspaceRoot,
+        workspaceLocation,
+      );
+
+      const rawPackageManifest = await getPackageManifest(
         completeWorkspacePath,
       );
+      const { name: packageName, version: packageVersion } =
+        validatePolyrepoPackageManifest(
+          rawPackageManifest,
+          completeWorkspacePath,
+        );
 
-    if (packageVersion === releaseVersion) {
+      if (packageVersion === releaseVersion) {
+        releaseNotes = releaseNotes.concat(
+          `## ${packageName}\n\n`,
+          await getPackageReleaseNotes(
+            releaseVersion,
+            repoUrl,
+            completeWorkspacePath,
+          ),
+          '\n\n',
+        );
+      }
+    }
+  } else {
+    for (const [packageName, value] of Object.entries(getUpdatedPackages())) {
+      const { path: packagePath } = value;
       releaseNotes = releaseNotes.concat(
         `## ${packageName}\n\n`,
         await getPackageReleaseNotes(
           releaseVersion,
           repoUrl,
-          completeWorkspacePath,
+          String(packagePath),
```

Comment:
Nit: I would expect this to be a string already. If it's not a string, what is it?

```suggestion
          packagePath,
```